spec/index.html

```diff
@@ -511,6 +515,8 @@ <h3>RDF Documents and Syntaxes</h3>
     and still the subject of group discussion. An alternative to conformance
     levels, "profiles", may be adopted instead, abandoned, or described in 
     another specification.</p>
+  <p class="issue" data-number="70">
```

Comment:
This is coming out as a URL of `.../issues70` not `.../issues/70`.

But putting in `data-number="/70"` making it "NAN" (!!!!)

The problem seems to be the processing and this chnage is correct, just not working because of elsewhere.